lib/cog/commands/alias/helpers.ex

```diff
@@ -101,7 +101,13 @@ defmodule Cog.Commands.Alias.Helpers do
     do: EctoJson.render(data)
 
   # Special errors that come when there are issues with the database.
-  defp db_errors(errors),
-    do: Enum.map_join(errors, "\n", fn({key, message}) -> "#{key} #{message}" end)
+  defp db_errors(errors) do
+    Enum.map_join(errors, "\n", fn
+      ({:name, "has already been taken"}) ->
```

Comment:
TIL: `:message`

It's a bit weird since we have to put the message on both models. But I think I agree with you that moving it to the model is best.